### ISSUE TITLE
Pidgin will not connect to slack

### ISSUE THREAD

**Issue by vanquishedangel** [dylex/slack-libpurple#177]
I feel like I am a newb at this but have used Linux for 20+ years lol. First off I refurbished and ancient Dell latitude D430 sub-notebook 2 gig ram with q4os 64 bit on it and am using it for work. Work uses slack and while slack works, I was looking for something more efficient for the computer.

I am using the version of pidgin that come with Debian bullseye which is 2.14.1-1 and have your plugin installed however when trying to connect to slack it said "unknown_method". I tried updating certificates, not sure I did it correctly, and I went to download a newer version of pidgin from their website but I can not for the life of me find where to download the source in their instructions.

I tried the token method but it then said invalid json response. I have read that you can switch to to "form data" instead of Json but I have no clue what that means.

Red Sentences in Debug:
(21:39:20) GLib-GObject: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(21:42:13) nss: partial certificate chain

**Comment by vanquishedangel** [comment]
I got it to work!

I trid your hack but due to my lack of experience I am not sure I did it correctly! I cloned the git to my home folder and on the web page selected "finduserhack". I went with the first one and copied the raw data for slack.auth.c and before building the pluging I copied it over the file I downloaded. I then built it following your instructions.

I did use the process to extract the API tokens from the browser but those did not work. I then looked up how to find the user ID in slack and copied that in the username field, and with my password, they all worked, all three of my slack workspaces worked!

Thank you so much for this plugin!
